### ISSUE TITLE
Crash when importing .glb with Static Lightmaps

### ISSUE THREAD

**Issue by lyuma** [godotengine/godot#62267]
### Godot version

4.0.alpha10

### System information

Windows 10

### Issue description

Importing Fox.glb with Static Lightmaps crashes. See attached project or try this file: [Fox.glb.gz](https://github.com/godotengine/godot/files/8946474/Fox.glb.gz)

I'm unable to get a stacktrace from the offiical Alpha 10 build. It simply prints:
```
ERROR: Cannot open file 'res://tp/new_theme.tres'.
   at: (scene/resources/resource_format_text.cpp:1412)
ERROR: Failed loading resource: res://tp/new_theme.tres. Make sure resources have been imported by opening the project in the editor at least once.
   at: (core/io/resource_loader.cpp:205)
WARNING: FBX file import is enabled, but no FBX2glTF path is configured. FBX files will not be imported.
     at: _editor_init (modules/gltf/register_types.cpp:92)
ERROR: Cannot open file 'res://.godot/imported/Fox.glb-b28195c156663faac07effc9017ee98e.scn'.
   at: (core/io/resource_format_binary.cpp:1125)
ERROR: Index p_index = 0 is out of bounds (size() = 0).
   at: get (./core/templates/cowdata.h:156)

[1]+  Illegal instruction     'C:\Users\crann\Downloads\Godot_v4.0-alpha10_win64.exe' -w -e
```

```
Godot Engine v4.0.alpha.custom_build.b78f7e274 - https://godotengine.org
WARNING: GENERAL - Message Id Number: 0 | Message Id Name: Loader Message
        ReadDataFilesInRegistry: Registry lookup failed to get layer manifest files.
        Objects - 1
                Object[0] - VK_OBJECT_TYPE_INSTANCE, Handle 1849560633888
     at: VulkanContext::_debug_messenger_callback (drivers\vulkan\vulkan_context.cpp:156)
Vulkan API 1.2.182 - Using Vulkan Device #0: NVIDIA - NVIDIA GeForce RTX 2080 Ti

ERROR: Cannot open file 'res://tp/new_theme.tres'.
   at: (scene\resources\resource_format_text.cpp:1412)
ERROR: Failed loading resource: res://tp/new_theme.tres. Make sure resources have been imported by opening the project in the editor at least once.
   at: (core\io\resource_loader.cpp:206)
WARNING: FBX file import is enabled, but no FBX2glTF path is configured. FBX files will not be imported.
     at: _editor_init (modules\gltf\register_types.cpp:92)
ERROR: Index p_index = 0 is out of bounds (size() = 0).
   at: CowData<struct Vector3>::get (S:\repo\godot-pr-pr\core/templates/cowdata.h:156)

================================================================
CrashHandlerException: Program crashed
Engine version: Godot Engine v4.0.alpha.custom_build (b78f7e274ee86350f20b6043da941d6781e9c9bd)
Dumping the backtrace. Please include this when reporting the bug on https://github.com/godotengine/godot/issues
[0] CowData<Vector3>::get (S:\repo\godot-pr-pr\core\templates\cowdata.h:156)
[1] Vector<Vector3>::operator[] (S:\repo\godot-pr-pr\core\templates\vector.h:92)
[2] ImporterMesh::lightmap_unwrap_cached (S:\repo\godot-pr-pr\scene\resources\importer_mesh.cpp:1057)
[3] ResourceImporterScene::_generate_meshes (S:\repo\godot-pr-pr\editor\import\resource_importer_scene.cpp:1644)
[4] ResourceImporterScene::_generate_meshes (S:\repo\godot-pr-pr\editor\import\resource_importer_scene.cpp:1722)
[5] ResourceImporterScene::_generate_meshes (S:\repo\godot-pr-pr\editor\import\resource_importer_scene.cpp:1722)
[6] ResourceImporterScene::_generate_meshes (S:\repo\godot-pr-pr\editor\import\resource_importer_scene.cpp:1722)
[7] ResourceImporterScene::import (S:\repo\godot-pr-pr\editor\import\resource_importer_scene.cpp:2076)
[8] EditorFileSystem::_reimport_file (S:\repo\godot-pr-pr\editor\editor_file_system.cpp:1914)
[9] EditorFileSystem::reimport_files (S:\repo\godot-pr-pr\editor\editor_file_system.cpp:2161)
[10] ImportDock::_reimport (S:\repo\godot-pr-pr\editor\import_dock.cpp:558)
[11] ImportDock::_reimport_attempt (S:\repo\godot-pr-pr\editor\import_dock.cpp:491)
[12] call_with_variant_args_helper<ImportDock> (S:\repo\godot-pr-pr\core\variant\binder_common.h:241)
[13] call_with_variant_args<ImportDock> (S:\repo\godot-pr-pr\core\variant\binder_common.h:351)
[14] CallableCustomMethodPointer<ImportDock>::call (S:\repo\godot-pr-pr\core\object\callable_method_pointer.h:105)
[15] Callable::call (S:\repo\godot-pr-pr\core\variant\callable.cpp:51)
[16] Object::emit_signalp (S:\repo\godot-pr-pr\core\object\object.cpp:1120)
[17] Object::emit_signal<> (S:\repo\godot-pr-pr\core\object\object.h:773)
[18] BaseButton::_pressed (S:\repo\godot-pr-pr\scene\gui\base_button.cpp:135)
[19] BaseButton::on_action_event (S:\repo\godot-pr-pr\scene\gui\base_button.cpp:176)
[20] BaseButton::gui_input (S:\repo\godot-pr-pr\scene\gui\base_button.cpp:67)
[21] Control::_call_gui_input (S:\repo\godot-pr-pr\scene\gui\control.cpp:940)
[22] Viewport::_gui_call_input (S:\repo\godot-pr-pr\scene\main\viewport.cpp:1302)
[23] Viewport::_gui_input_event (S:\repo\godot-pr-pr\scene\main\viewport.cpp:1610)
[24] Viewport::push_input (S:\repo\godot-pr-pr\scene\main\viewport.cpp:2742)
[25] Window::_window_input (S:\repo\godot-pr-pr\scene\main\window.cpp:974)
[26] call_with_variant_args_helper<Window,Ref<InputEvent> const &,0> (S:\repo\godot-pr-pr\core\variant\binder_common.h:236)
[27] call_with_variant_args<Window,Ref<InputEvent> const &> (S:\repo\godot-pr-pr\core\variant\binder_common.h:351)
[28] CallableCustomMethodPointer<Window,Ref<InputEvent> const &>::call (S:\repo\godot-pr-pr\core\object\callable_method_pointer.h:105)
[29] Callable::call (S:\repo\godot-pr-pr\core\variant\callable.cpp:51)
[30] DisplayServerWindows::_dispatch_input_event (S:\repo\godot-pr-pr\platform\windows\display_server_windows.cpp:2077)
[31] DisplayServerWindows::_dispatch_input_events (S:\repo\godot-pr-pr\platform\windows\display_server_windows.cpp:2042)
[32] Input::_parse_input_event_impl (S:\repo\godot-pr-pr\core\input\input.cpp:662)
[33] Input::flush_buffered_events (S:\repo\godot-pr-pr\core\input\input.cpp:888)
[34] DisplayServerWindows::process_events (S:\repo\godot-pr-pr\platform\windows\display_server_windows.cpp:1784)
[35] OS_Windows::run (S:\repo\godot-pr-pr\platform\windows\os_windows.cpp:777)
[36] widechar_main (S:\repo\godot-pr-pr\platform\windows\godot_windows.cpp:175)
[37] _main (S:\repo\godot-pr-pr\platform\windows\godot_windows.cpp:197)
[38] main (S:\repo\godot-pr-pr\platform\windows\godot_windows.cpp:211)
[39] WinMain (S:\repo\godot-pr-pr\platform\windows\godot_windows.cpp:225)
[40] __scrt_common_main_seh (D:\agent\_work\9\s\src\vctools\crt\vcstartup\src\startup\exe_common.inl:288)
[41] BaseThreadInitThunk
-- END OF BACKTRACE --
================================================================
```

The .import file has:
```
nodes/root_type="Node3D"
nodes/root_name="Scene Root"
nodes/root_scale=0.1
meshes/ensure_tangents=true
meshes/generate_lods=true
meshes/create_shadow_meshes=true
meshes/light_baking=2
meshes/lightmap_texel_size=0.1
skins/use_named_skins=true
animation/import=true
animation/fps=30
import_script/path=""
_subresources={}
```

### Steps to reproduce

1. Open the project
2. It should crash. if not, select Fox.glb
3. Make sure the lightmap mode is set to Static Lightmaps
4. Click Reimport
5. the editor will crash.

### Minimal reproduction project

[static_lightmaps_crash.zip](https://github.com/godotengine/godot/files/8946469/static_lightmaps_crash.zip)

**Comment by lyuma** [comment]
@fire linked me to this change:
https://github.com/V-Sekai/godot/tree/lightmap-errors

I opened a draft PR from this branch, but I have not yet tested it:
https://github.com/godotengine/godot/pull/62268

**Comment by lyuma** [comment]
This model has neither normals nor tangents:
```
"meshes":[{"name":"fox1","primitives":[{"attributes":{"POSITION":0,"TEXCOORD_0":1,"JOINTS_0":2,"WEIGHTS_0":3},"material":0}]}]
```

#62268 appears to prevent the crash. However, no message is printed during import. I wonder if it would be good to generate normals on import if the user has "Ensure Tangents" selected (how can you ensure tangents if no normals exist?)

Also, if I manually select Unwrap UV2 from the mesh menu, I get some generic "mesh might not be manifold" error dialog, but the actual error "  Normals are required for lightmap unwrap." is printed out.

Blender does add normals in exports, so this case might be rare.

**Comment by lyuma** [comment]
It was fixed by #62268 - just tested the MRP and it works now.
